### ISSUE TITLE
Support for Github "Alerts"

### ISSUE THREAD

**Issue by Sajjon** [jgm/pandoc#10109]
**Describe your proposed improvement and the problem it solves.**
Seems this (great) tool does not support [Github Markdown "alerts"](https://github.com/orgs/community/discussions/16925), at least not without any extra options?

**Describe alternatives you've considered.**
N/A

**Comment by Sajjon** [comment]
This is perhaps a duplicate of https://github.com/jgm/pandoc/issues/9716

**Comment by jgm** [comment]
No, it's not a duplicate of #9716, which concerns pandoc's `markdown` and not `gfm`.  GitHub alerts *are* supported in `gfm`.

```
pandoc -f gfm -t native
> [!CAUTION]
> You **MUST** use the Rust programming language.
^D
[ Div
    ( "" , [ "caution" ] , [] )
    [ Div ( "" , [ "title" ] , [] ) [ Para [ Str "Caution" ] ]
    , Para
        [ Str "You"
        , Space
        , Strong [ Str "MUST" ]
        , Space
        , Str "use"
        , Space
        , Str "the"
        , Space
        , Str "Rust"
        , Space
        , Str "programming"
        , Space
        , Str "language."
        ]
    ]
]
```

The issue is that the writers don't by default do any special styling for this structure.  See #9821 for a filter that may help with PDF output.

**Comment by jgm** [comment]
For background see also
https://github.com/jgm/commonmark-hs/issues/132#issuecomment-1856751436
